### DISCUSSION TITLE
The WiFiRubbernuggetInvader¯\_( ^ ° ^ )_/¯

### DISCUSSION THREAD

**Discussion by DedsecEnzo** [DevKitty-io/USB-Nugget#18] [Ideas]
Hey my idea would be to connect the new rubbernugget software with the nugget invader software and maybe if that possible to be able on the Esp32 you can choose in a small Menu between a wifi attack or a usb attack. I hope everyone can understand me because my English is not the best^-^

Sincerely, Enzo

**Comment by skickar** [comment]
Hi Enzo!

Currently, the esp32s2 does not support the same attacks as the invader, because it can't send deauth packets yet. This might be possible in the future, though, in which case it would be easy to do so.

This is possible, and would add Wi-Fi spoofing and deauthing. It's not too hard to make an ESP32s2 talk to an ESP8266. I think this design uses an esp32 and an ESP8266: https://github.com/justcallmekoko/ESP32Marauder

**Reply by DedsecEnzo** [reply]
Hey Kody(^-^)

If you say not yet, do you think it would be worth considering or is there no prospect of upgrading the NuggetInvader for the Esp32
I'm not exactly a professional like you :D Connecting the whole Rubbernugget project with the NuggetInvader is far beyond my knowledge. 🤭
